tests/vm_test.go

```diff
@@ -169,7 +170,28 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		type vmiBuilder func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume)
 
 		newVirtualMachineInstanceWithFileDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-			return tests.NewRandomVirtualMachineInstanceWithFileDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), testsuite.GetTestNamespace(nil), corev1.ReadWriteOnce)
+			sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
```

Comment:
https://github.com/kubevirt/kubevirt/blob/5ecef19996b770412a531bf2c46cbfadecc54a8a/tests/utils.go#L204-L206

Do we need to keep the `Skip` from `NewRandomVirtualMachineInstanceWithFileDisk`?

Reply:
Added

Reply:
where?

Reply:
Added the Expect line for CDI instead of skip.